src/components/Footer.tsx

```diff
@@ -1,54 +1,62 @@
-import { Box, Center, HStack, Stack } from '@chakra-ui/react'
+import { Box, Center, HStack, Stack, useColorModeValue } from '@chakra-ui/react'
 import Image from 'next/image'
 
 import poweredByVercelLogo from '@app-public/powered-by-vercel.svg'
 import { REPO_URL } from '~/common'
 import FluidContainer from '~/components/FluidContainer'
 import Link from '~/components/Link'
 
-const Footer = () => (
```

Comment:
Although it seems there are tons of changes on this component, it's just a reformating after switching from arrow function with implicit return (i.e. `() => <Foo />`) to arrow function with explicit return (i.e. `() => { return <Foo /> }`)